spec/features/eligible_voters_spec.rb

```diff
@@ -4,7 +4,11 @@
 require "csv"
 
 RSpec.feature "Eligible voters csv" do
-  before { Timecop.freeze(Time.utc(2016, 5, 5, 10, 5, 3)) }
+  before {
+    Timecop.freeze(Time.utc(2016, 5, 5, 10, 5, 3))
+    sign_in board_member
+  }
```

Comment:
Two lines means one of your other cop PRs will want to turn it into a "proper" `do` block I think